front/src/modules/ui/table/options/components/TableViewsDropdownButton.tsx

```diff
@@ -60,6 +60,7 @@ const StyledViewIcon = styled(IconList)`
 const StyledViewName = styled.span`
   display: inline-block;
   max-width: 200px;
+  min-width: 118px;
```

Comment:
Ok for the fix! I still think it's not the right way to deal with Dropdown components and that we need to provide an API to our users that will take care of these width questions